### PR TITLE
Math Magicians Full website

### PR DESCRIPTION
**Pull request by zakiyousufi**
computer

### **_Attention Please_**
_**In this Project, I have used**_
 - CSS
 - JavaScript
 - React
 - ES6 Modules
 - following linters standards.
 - following Github flows standards.
 - following git flows standards.
 - webpak

**In PR [#](https://github.com/zakiyousufi/math-magicint/pull/6)6**

- [x] - I have designed the Math Magicians full-website 
- [x] - I have used `Routes` and `Route` to swich the pages 
- [x] - My website has 3 pages with a navbar  

Have fun 😊😉

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter as Router } from 'react-router-dom';
```
Comment (ShoiraTa) [inline src/index.js:3]: - [ ] When I run the project, I was getting an error that said:  `Can't resolve 'react-router-dom'`, then, when I run `npm install react-router-dom --save` everything worked fine. Please add `react-router-dom`  to package.json by running `npm install react-router-dom --save`

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @zakiyousufi ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (zakiyousufi) [reply src/index.js:3]: OK thanks for your Help

**Review by Reem-lab — APPROVED** [review]
# Hi @zakiyousufi,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality and the design✅
📌 Good commit messages ✅
📌 Professional README ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**